docs/interviewing_options.md

```diff
@@ -1 +1,134 @@
+  at a typical job
+- Can be done during non work hours allowing candidates more freedom
+- Less anxiety involved for candidate as no one is watching them
+- Certain types of tests might be conducive to automated testing to check
+  results and validity
+- Again this can be a cheap way to screen candidates as they don't need to come
+  on-site, and act as a stopper for those who are not super serious about
+  actually interviewing
+- Removes the power imbalance that can occur in whiteboard style interview.
+
+#### Cons
+
+- Possibility of candidates plagiarizing work
+- Can bias against people whom have family commitments (caregivers,
+  single parents, etc.) depending on the level of effort needed
+- Again can unintentionally discriminate based on age (favouring those whom
+  might have more free time and can iterate on better solutions)
+- Depending on the complexity of the assignment it may require time for someone
+  to grade and properly evaluate. Depending on the volume of candidates this
+  can be a huge problem.
+
+
+## Bring your own personal code or project
+
+Bringing your own personal code or projects is another method similar to a take
+type of test. The main difference here is that this is something the candidate
+has built vs being told to do. This could be a open source project which they
+have contributed to or maybe a smaller hobby type project. During the interview
+process
+
+#### Pros
+
+- Allows the candidate to work on their own time, space, environment setup, etc.
+- Can be done during non work hours allowing candidates more freedom
+- Less anxiety for the candidate
+- Removes the power imbalance that can occur in whiteboard style interview.
+- Shows the candidates passions outside their core work or job.
+
+#### Cons
+
+- Possibility of candidates plagiarizing work
+- Not everyone has hobby projects which they can show off
+- Again this could indirectly discriminate against some candidates who do not
+  have the time outside of work (family, children, etc.)
+- project may not be relevant to the job at hand therefore making it hard to
+  evaluate the candidates skill set
+
+
+
+## Automated coding challenge
+
+Typical websites which can ask a variety of coding question with "correct"
+answers. Can be many question or sometimes a single longer one. These sites will
+have a online editor (for the language of choice) and a timer. Sometimes the
+questions can be multiple choice, coding questions with a set of test cases
+which must pass, or combination of both.
+
+#### Pros
+
+- Allows the candidate to work on their own time, space, environment setup, etc.
+- Can be done during non work hours allowing candidates more freedom
+- Cheap and quick screening method.
+- Fully automated.
+
+#### Cons
+
+- Possibility of candidates plagiarizing work may still exist.
+- Again this could indirectly discriminate against some candidates who do not
+have the time outside of work (family, children, etc.)
```

Comment:
I like the wording of this bullet better than what you had above and better than what I suggested. I would used this wording, but add ... (family, children, caregiver, etc.)

And for that matter, should we add religious commitments? If the interview came at the wrong time of year, it could be really hard. 

And community commitments or is that a stretch? (I am thinking it is stretch.)

Reply:
Religious accommodations makes sense.